pkgs/by-name/va/vapoursynth-znedi3/package.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation {
   installPhase = ''
     runHook preInstall
 
-    install -D -t $out/lib/vapoursynth vsznedi3${stdenv.hostPlatform.extensions.sharedLibrary}
```

Comment:
Is it `.so` instead of `.dylib` even on darwin?

Reply:
The file extension is hardcoded in the Makefile: https://github.com/sekrit-twc/znedi3/blob/master/Makefile#L90